applications/ChatGPT/chatgpt/trainer/sft.py

```diff
@@ -0,0 +1,97 @@
+from abc import ABC
+
+import loralib as lora
+import torch
+from chatgpt.dataset import SFTDataset, SFTDistributedDataset
+from chatgpt.models.loss import GPTLMLoss
+from torch.optim import Adam, Optimizer
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+import torch.distributed as dist
+from .strategies import Strategy
+from .utils import is_rank_0
+from colossalai.logging import get_dist_logger
+
+
+class SFTTrainer(ABC):
+    """
+        Trainer to use while training reward model.
+
+    Args:
+        model (torch.nn.Module): the model to train
+        strategy (Strategy): the strategy to use for training
+        optim(Optimizer): the optimizer to use for training
+        train_dataset (RewardDataset): the dataset to use for training
```

Comment:
SFT Dataset？

Reply:
Yes, I borrowed some code from the origin project, and I forgot to check the comment code...